llvm/docs/TestSuiteGuide.md

```diff
@@ -266,7 +269,7 @@ Example usage:
 
   ```bash
   % test-suite/utils/compare.py -m compile_time baseline.json
-  % test-suite/utils/compare.py -m size.__text baseline.json
+  % test-suite/utils/compare.py -m size..text baseline.json
```

Comment:
Not sure this should change. Apparently, they’re both correct, depending on the platform. https://github.com/llvm/llvm-test-suite/blob/47a750394dfa29d10cdb515132a525d3e8fa3118/litsupport/modules/codesize.py#L41

Reply:
i see. i'll remove this.